src/plugins/LADTable/components/LADRow.vue

```diff
@@ -73,11 +79,22 @@ export default {
                 .request(this.domainObject, {strategy: 'latest'})
                 .then((array) => this.updateValues(array[array.length - 1]));
 
+        },
+        getDomainObjectPath(event) {
+            event.preventDefault();
+
+            this.openmct.objects.getOriginalPath(this.keyString).then((path) => {
```

Comment:
Unfortunately this can't be the original path, it has to be the contextual path of this object instance. It will have to be passed down from `LADTable.vue`. 

We have an open question right now about how views resolve the path of an object, and we don't have a good answer. The obvious solution is to have it passed into the `.view()` method of view provider (could be an optional argument), which can then pass it down to any subordinate components. The objectPath passed to `.view()` will need to be provided by the component calling it, which is typically `ObjectView.vue`. It will need to be passed to `ObjectView` from its parent - either `Layout.vue` which gets it from the navigated path, or `SubObjectView.vue` which gets it by appending its `domainObject` to the `objectPath` provided by its parent (and so on up the chain).

I'd like to get your thoughts though? Is there a better / easier way of doing this?

In any case, I don't want to commit code right now that we know is wrong and may cause bugs at some point.

Reply:
I agree with passing down the correct context path from object view. But for this use case, I was thinking we do not need to have a context menu drop down in the Historical View, thus defeating the purpose of passing in a correct contextual path. Since users do not need to be able to modify the contents of the historical view, is it ok if we didnt pass a contextual path?